app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+    <%= link_to  "Add a recipe", new_recipe_path, class:"btn btn-primary mb-4" %>
+    <% @recipes.each do |recipe| %>
+    <ul class="list-group">
+       <li class="list-group-item d-flex justify-content-between mb-4">
+        <div>
+          <h2><%= recipe.name %></h2>
+          <% if can? :destroy, recipe %>
+          <div class="d-flex">
+            <%= link_to  "DETAILS", recipe_path(recipe), class:"btn btn-primary me-4" %>
+            <%= link_to "REMOVE", recipe, data: { 'turbo-method': :delete, 'turbo-confirm': 'Are you sure?' }, class:"btn btn-danger" %>
+          </div>
+          <% end %>
+        </div>

```

Comment:
Hello @KDlamini, I have the list of the public recipes at this route: `http://localhost:3000/public_recipes`

Reply:
@ridwanediallo Okay, in that case kindly make sure that there's a button for it so that it is easily navigable as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/194718038-2671f9db-b83e-4cfd-8e4b-08b8b5163915.png)